curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/663620974eb4a0369d76b6b8.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 663620974eb4a0369d76b6b8
+title: Task 17
+challengeType: 22
+dashedName: task-17
+---
+
+<!--
+AUDIO REFERENCE: 
+Sarah: I think so. Why don't we check out some of these tools to see if they can help us?
```

Comment:
I wonder if it would be better to use `these` instead of `those`.

Reply:
@a2937 Do you mean `these` is better? Yes, that's exactly the word used in this task:)

Reply:
I think both these and those work here. Since `these` is used in the audio, we should probably stick with that.